.github/workflows/snyk.yaml

```diff
@@ -6,6 +6,9 @@ on:
     branches:
       - master
       - release-*
+  pull_request:
+    branches:
+      - master
 
```

Comment:
what about this empty line? should we add `release-*` here?

Reply:
I will need to revert this before merge. The only way this PR is working is because it is in the rook repo itself, and not from a fork.